### DISCUSSION TITLE
[Feature] Folder structure. Show pictures and videos from this user on your timeline

### DISCUSSION THREAD

**Discussion by fjaviys** [immich-app/immich#14952] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi!
The admin has the ability to show folder structure on external mounted storage, but when you share this external storage to users, those can see the timeline of the shared content, but can't navigate the folder structure although the option is checked. When go to 'folder' link on the sidebar, it shows empty.
Screenshot of the user account on Folder view:

![image](https://github.com/user-attachments/assets/2df3fbcf-d4fe-4ff5-8c25-5599496a07c6)

I think it will be interesting to enable this feature.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
To be clear, do you mean showing partner-shared assets in the folder view?

**Reply by fjaviys** [reply]
exactly! Thnk you for the aproach

**Comment by bo0tzz** [comment]
#1779
